torchci/lib/bot/cliParser.ts

```diff
@@ -34,7 +50,13 @@ mergeOption.add_argument("-f", "--force", {
 
 // Revert
 const revert = commands.add_parser("revert", {
-  help: "Revert a merged PR",
+  help: "Revert a PR",
+  description: `Revert a merged PR. This requires that you are a Meta employee.
+
+Example:
+  @pytorchbot revert -m="This is breaking tests on trunk. hud.pytorch.org/" -c=nosignal
```

Comment:
Should we really hint people at linking to the hud instead of a specific job?

Reply:
mm yeah I'll make a change